src/cmd/cli/command/compose_test.go

```diff
@@ -0,0 +1,77 @@
+package command
+
+import (
+	"testing"
+
+	compose "github.com/compose-spec/compose-go/v2/types"
+	assert "github.com/stretchr/testify/assert"
+)
+
+func TestIsOnlyManagedResources(t *testing.T) {
+
+	t.Run("no services", func(t *testing.T) {
+		project := &compose.Project{}
+		project.Services = make(map[string]compose.ServiceConfig)
+
+		assert.Equal(t, true, IsOnlyManagedResources(project))
```

Comment:
downside of this is the generic "expect X got Y" message without context. Copilot will complete the `if` for you with a better message.